docs/extend/modular-resources/create/_index.md

```diff
@@ -515,6 +515,201 @@ Expand the [Go module code](#code-a-main-entry-point-program) to view <file>main
 You need to ensure any dependencies for your module (including the Python or Go [Viam SDK](/program/)) are installed as well.
 Your executable will be run by `viam-server` as root, so dependencies need to be available to the root user.
 
+## Create a custom module using the Viam CLI
+
+{{% alert title="Beta Notice" color="note" %}}
+This feature is in beta, and may not be suitable for production use.
+{{% /alert %}}
+
+You can use the [Viam CLI](/manage/cli/) to create and upload your own custom modules to the Viam Registry.
+With the CLI, you can create a new module from a template, register your module with the Viam Registry, and share it with other Viam users.
+You can also create a private module that is shared only within your [organization](/manage/fleet/organizations/). For more information, see the [`viam module` command](/manage/cli/#module).
+
+1. First, [install the Viam CLI](/manage/cli/#install) and [authenticate](/manage/cli/#authenticate) to Viam.
+
+1. Next, choose your custom module name and create a new module template using the `viam module create` command:
+
+   - To create a *public* module that will be visible to all Viam users, provide a unique [namespace](/extend/modular-resources/key-concepts/#namespace) when creating your module:
+
+      ``` sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+      viam module create --name <your-module-name> --namespace <your-unique-namespace>
+      ```
+
+   - To create a *private* module that will only be accessible to users within your [organization](/manage/fleet/organizations/), provide your organization ID when creating your module:
+
+      ``` sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+      viam module create --name <your-module-name> --org-id <org-id>
+      ```
+
+      You can find your organization ID by navigating to the [the Viam app](https://app.viam.com), selecting your user account in the upper-right corner, and clicking **Settings** from the drop down menu.
+
+   This command creates a new `meta.json` file in your current working directory, which serves as a template on which to base your custom configurations.
+
+1. Edit the newly-created `meta.json` file, and provide the required configuration information for your custom module by filling in the following fields. The `name` and `visibility` fields are pre-populated:
+
+<table>
```

Comment:
I can't quite get this table to render intended underneath the step above. I'll sort this out tomorrow with our resident table-whiz Naomi!